font-changer.py

```diff
@@ -0,0 +1,29 @@
+import sys
+import re
+import os
+
+file_to_change = "src/index.html"
+
+output_file = "src/test.html"
```

Comment:
What is the use case for this, making a file test.html could break the extension or not have an effect on the UI

Reply:
I can change it, originally meant to tell the script to not override the previous file if you didn't want to. I think the opposite of the way it does now is better